Adding Speakers to the Feed.

punkcoder opened the pull request:
- Added functionality to 'switch email' boxes
- Worked on moving the helper scripts to helperscripts/
- generateemail.py now compiles all of the data from email, speakers, and sponsors

punkcoder committed:
Adding speakers

punkcoder committed:
Adding all of the speakers with formatting

kernelmethod committed:
Change Kurt's name to uppercase.

kernelmethod commented:
Looks good for the most part. I think we should try to add Amie and the keynote before we update the site with this though.

kernelmethod commented:
Also -- I should've done this a while ago, but I'm going to look into whether there's an easier way to generate these templates. I just committed a small change to Kurt's bio (literally just making his name uppercase) and it, uh, was not super fun. :upside_down_face:

punkcoder commented:
@kernelmethod all of that Information is being populated directly from sessionize, we need to get Amie into Sessionize and get Kurt's bio updated. When we do that I can rebuild them quickly.

punkcoder commented:
@kernelmethod from src run ./helperscripts/generatespeakerhtml.py and you should be able to generate the speakers, then running ./helperscript/generateemail.py will rebuild the email boxes.

punkcoder committed:
Adding fix for name not being capitolized

punkcoder committed:
Merge branch 'dev/speakers' of github.com:BSidesBoulder/site.bsidesboulder2021 into dev/speakers

punkcoder commented on src/helperscripts/generatespeakerhtml.py:
```diff
@@ -27,6 +27,12 @@
     speaker = speaker_pre
     speaker['description'] = sessionLookup[speaker['sessions'][0]['id']]
+
+    #srsly hate hacks like this
```
@kernelmethod this was the fastest way to fix this permanently....

punkcoder reviewed (commented):
Okay at this point we should be good to go... if I can get the approve thumbs up then it will go live.

kernelmethod committed:
Small fixes to the generatespeakerhtml.py script.

- Instead of having a special case for Kurt, split up the components of
  the name and call '.title()' on all of them.
- Use base64.b64encode instead of base64.encodestring as the latter is
  deprecated (and adds unnecessary newlines).

kernelmethod commented on src/helperscripts/generatespeakerhtml.py:
```diff
@@ -0,0 +1,47 @@
+    speaker = speaker_pre
+    speaker['description'] = sessionLookup[speaker['sessions'][0]['id']]
+
+    # Capitalize name if necessary
+    name_components = speaker['fullName'].split(" ")
+    speaker['fullName'] = " ".join(c.title() for c in name_components)
+    print(speaker['fullName'])
```
Is this any better? Only slightly less hacky than your original workaround I'm afraid
